substrate/frame/staking/src/mock.rs

```diff
@@ -291,6 +290,9 @@ impl OnStakingUpdate<AccountId, Balance> for EventListenerMock {
 	}
 }
 
+// Disabling threshold for `UpToThresholdDisablingStrategy`
+pub(crate) const DISABLING_THRESHOLD_FACTOR: usize = 3;
```

Comment:
This const exists only because I need it in a test [here](https://github.com/paritytech/polkadot-sdk/blob/68585e13cb3c83107275f7d756f7dcc55339709f/substrate/frame/staking/src/tests.rs#L3413).

`UpToThresholdDisablingStrategy` is supposed to be generic and we decided to follow the pattern where the disabling threshold factor is passed as a generic parameter. For some reason the compiler is not happy with `UpToThresholdDisablingStrategy::disable_threshold(..)`:
```
error[E0282]: type annotations needed
    --> substrate/frame/staking/src/tests.rs:3413:5
     |
3413 |                 UpToThresholdDisablingStrategy::disable_threshold(Session::validators().len()),
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot infer the value of the const parameter `DISABLING_THRESHOLD_FACTOR` declared on the struct `UpToThresholdDisablingStrategy`
     |
help: consider specifying the generic argument
     |
3413 |                 UpToThresholdDisablingStrategy::<DISABLING_THRESHOLD_FACTOR>::disable_threshold(Session::validators().len()),
     |                                               ++++++++++++++++++++++++++++++

For more information about this error, try `rustc --explain E0282`.
error: could not compile `pallet-staking` (lib test) due to previous error

```

So to avoid discrepancies in the future I introduced this constant and use it on both places.